python/taichi/ui/scene.py

```diff
@@ -13,6 +13,27 @@
 normals_field_cache = {}
 
 
+class DisplayMode_:
+    Fill = _ti_core.DisplayMode.Fill if _ti_core.GGUI_AVAILABLE else None
+    Line = _ti_core.DisplayMode.Line if _ti_core.GGUI_AVAILABLE else None
+    Point = _ti_core.DisplayMode.Point if _ti_core.GGUI_AVAILABLE else None
+
+    @staticmethod
+    def __call__(mode: int):
```

Comment:
This doesn't make sense hmmm: it seems like you never used this call method at all...

Reply:
I would like to use display mode through an int value with imgui to control it every frame. As below:

```python
display_mode = 0
def show_options(): # You can choose what display mode you like in realtime
    global display_mode    
    window.GUI.begin("Display Panel", 0.05, 0.1, 0.2, 0.15)
    # display_mode (0:Fill Mode, 1:Line Mode, 2:Point Mode)
    display_mode = window.GUI.slider_int("display_mode", display_mode, 0, 2)
    window.GUI.end()

scene.mesh_instance(vertices, indices, instance_count = 100 ,display_mode=ti.ui.DisplayMode(display_mode)))
```
Here, I want to use `ti.ui.DisplayMode(display_mode)` instead of `if else` blablabla.

Reply:
as discussed offline, a more user friendly implementation should show verbose enum name in strings instead of ints. e.g https://github.com/ocornut/imgui/blob/master/imgui_demo.cpp#L738-L747